pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -634,6 +634,12 @@ stdenvNoCC.mkDerivation {
     + optionalString ((targetPlatform ? gcc.cpu) && (isClang || !(targetPlatform.isDarwin && targetPlatform.isAarch64))) ''
       echo "-mcpu=${targetPlatform.gcc.cpu}" >> $out/nix-support/cc-cflags-before
```

Comment:
Yes it should replace it too, ideally.

Reply:
Yes, I just didn't strip out the way things are done with `gcc.tune` because I didn't want to break too much. My initial goal was to just get this mechanism to work.

Reply:
Unlike getting rid of `architecture.nix`, I am fine saving that for later :).

Reply:
Yeah, that was my plan. Maybe add a couple warnings that the file is up for removal so that we can give people a buffer